src/community/pom.xml

```diff
@@ -73,7 +66,6 @@
           <descriptor>release/ext-jdbcstore.xml</descriptor>
```

Comment:
You should remove ext-python.xml just above as well

Reply:
Done, moved it down to the appropriate profile.